makefiles/logcleaner.mk

```diff
@@ -0,0 +1,39 @@
+ifneq ($(PROCURSUS),1)
+$(error Use the main Makefile)
+endif
+
+ifeq (,$(findstring darwin,$(MEMO_TARGET)))
+
+STRAPPROJECTS += logcleaner
```

Comment:
I don't think this should be a ``STRAPPROJECT``, but if that's the case, you have to do more (add it as part of the bootstrap, or something like... someone can help you out; it's a minor step that you missed), otherwise, use ``SUBPROJECTS``. Also, fix the indentation of this variable.

Reply:
oh, i just copy-pasted `shshd.mk` and had no clue what `STRAPPROJECTS` was

Reply:
Definitely make this a `SUBPROJECT`.